pkgs/servers/nosql/mongodb/v4_0.nix

```diff
@@ -14,4 +14,5 @@ in buildMongoDB {
   patches =
     [ ./forget-build-dependencies.patch ./mozjs-45_fix-3-byte-opcode.patch ]
     ++ lib.optionals stdenv.isDarwin [ ./asio-no-experimental-string-view.patch ];
+  broken = true;
```

Comment:
I would leave v4.0, per https://www.mongodb.com/support-policy/lifecycles is not yet EOL'ed.